deepmd/pt/model/descriptor/sezm.py

```
@@ -746,21 +821,35 @@ def __init__(
+                    )
+                )
+            else:
+                self.gie_zonal_wigner_calc = None
         else:
             self.gie = None
+            self.gie_zonal_wigner_calc = None
 
         blocks: list[SeZMInteractionBlock] = []
-        for block_idx, (l_b, m_b) in enumerate(zip(self.l_schedule, self.m_schedule)):
+        for block_idx, (l_b, node_l_b, m_b) in enumerate(
+            zip(self.l_schedule, self.node_l_schedule, self.m_schedule)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)